pgo/cmd/backup.go

```diff
@@ -82,6 +83,7 @@ func init() {
 	backupCmd.Flags().StringVarP(&BackupOpts, "backup-opts", "", "", "The options to pass into pgbackrest.")
 	backupCmd.Flags().StringVarP(&Selector, "selector", "s", "", "The selector to use for cluster filtering.")
 	backupCmd.Flags().StringVarP(&PVCName, "pvc-name", "", "", "The PVC name to use for the backup instead of the default.")
+	backupCmd.Flags().StringVarP(&PGDumpDB, "pgdump-db", "", "postgres", "The name of the database pgDump will backup.")
```

Comment:
I think `--database` (and even with a `-d` as the secondary one) would be fine.

Reply:
ok, that sounds good to me, I'll make that change. Thanks.